submissions/akulahere/html-css-popup/index.html

```diff
@@ -0,0 +1,191 @@
+                                Map
+                            </p>
+                        </a>
+                    </li>
+                    <li class="pop-up-item">
+                        <a href="#">
+                            <img src="img/files.png">
+                            <p>
+                                Files
+                            </p>
+                        </a>
+                    </li>
+                    <li class="pop-up-item">
+                        <a href="#">
+                            <img src="img/contacts.png">
+                            <p>
+                                Contacts
+                            </p>
+                        </a>
+                    </li>
+                    <li class="pop-up-item">
+                        <a href="#">
+                            <img src="img/tanslation.png">
+                            <p>
+                                Translate
+                            </p>
+                        </a>
+                    </li>
+                    <li class="pop-up-item">
+                        <a href="#">
+                            <img src="img/videos.png">
+                            <p>
+                                Videos
+                            </p>
+                        </a>
+                    </li>
+                    <li class="pop-up-item">
+                        <a href="#">
+                            <img src="img/search.png">
+                            <p>
+                                Search
+                            </p>
+                        </a>
+                    </li>
+                    <li class="pop-up-item">
+                        <a href="#">
+                            <img src="img/photos.png">
+                            <p>
+                                Photo
+                            </p>
+                        </a>
+                    </li>
+                    <li class="pop-up-item">
+                        <a href="#">
+                            <img src="img/apps.png">
+                            <p>
+                                App
+                            </p>
+                        </a>
+                    </li>
+                    <li class="pop-up-item">
+                        <a href="#">
+                            <img src="img/documents.png">
+                            <p>
+                                Documents
+                            </p>
+                        </a>
+                    </li>
+                    <li class="app-menu-more">
+                        <input type = "checkbox" id = "show-more" class="show-more-cb">
+                        <label for = "show-more" class = "show-more-label">
+                            Show more
+                        </label>
+                        <hr>
+                        <ul class = "hidden-elements"?>
+                            <li class = "hidden-item">
```

Comment:
I think, that element `?` should be delete.

Reply:
deleted